src/runtime_init_memory.js

```diff
@@ -29,6 +21,11 @@ if (ENVIRONMENT_IS_PTHREAD) {
   } else
 #endif
   {
+    {{{ makeModuleReceiveWithVar('INITIAL_MEMORY', undefined, INITIAL_MEMORY) }}}
+
+    #if ASSERTIONS
+    assert(INITIAL_MEMORY >= {{{STACK_SIZE}}}, 'INITIAL_MEMORY should be larger than STACK_SIZE, was ' + INITIAL_MEMORY + '! (STACK_SIZE=' + {{{STACK_SIZE}}} + ')');
+    #endif
```

Comment:
Indentation looks wrong here.
```suggestion
#if ASSERTIONS
    assert(INITIAL_MEMORY >= {{{STACK_SIZE}}}, 'INITIAL_MEMORY should be larger than STACK_SIZE, was ' + INITIAL_MEMORY + '! (STACK_SIZE=' + {{{STACK_SIZE}}} + ')');
#endif
```